ext/storage_sources/s3_store/s3_storage_source.cpp

```diff
@@ -238,49 +259,36 @@ S3CustomizeFileSystem(WT_STORAGE_SOURCE *storageSource, WT_SESSION *session, con
 
     /* TODO: Move these into tests. Just testing here temporarily to show all functions work. */
```

Comment:
The final change for this PR will remove this test block - when all tests are green.